man/uwsm.1.scd

```diff
@@ -388,24 +388,27 @@ when logged in on VT 1:
 
 ```
 	if uwsm may-start && uwsm select; then
-		exec uwsm start default
+		exec systemd-cat -t uwsm-start uwsm start default
 	fi
 ```
 
 This just starts a specific compositor depending on foreground VT:
 
 ```
 	if uwsm may-start 1; then
-		exec uwsm start sway.desktop
+		exec systemd-cat -t uwsm-start uwsm start sway.desktop
 	elif uwsm may-start 2; then
-		exec uwsm start labwc.desktop
+		exec systemd-cat -t uwsm-start uwsm start labwc.desktop
 	fi
 ```
 
 Using "*uwsm may-start*" as a condition is *essential*, not only to prevent
 accidental startup attempts where they are not expected, but also since startup
 involves sourcing shell profile, which might lead to nasty loops.
 
+*systemd-cat -t uwsm-start* runs the command given to it with its stdout
+connected to the systemd journal, tagged with identifier "uwsm-start".
+
```

Comment:
might be a good idea to direct user towards `systemd-cat(1)` for things like priority options and mention `systemd-cat(1)` in `SEE ALSO`